src/Catalyst.Core.Modules.Kvm/DeltaExecutor.cs

```diff
@@ -217,43 +205,21 @@ private void Execute(PublicEntry entry, StateUpdate stateUpdate, ITxTracer txTra
             var gasLimit = entry.GasLimit;
             var intrinsicGas = CalculateIntrinsicGas(entry, spec);
 
-            if (_logger.IsEnabled(LogEventLevel.Verbose))
-            {
-                _logger.Verbose("Executing entry {entry}", entry);
-            }
+            if (_logger.IsEnabled(LogEventLevel.Verbose)) _logger.Verbose("Executing entry {entry}", entry);
 
-            if (!ValidateSender(entry, env, txTracer))
-            {
-                return;
-            }
+            if (!ValidateSender(entry, env, txTracer)) return;
 
-            if (!ValidateIntrinsicGas(entry, env, intrinsicGas, txTracer))
-            {
-                return;
-            }
+            if (!ValidateIntrinsicGas(entry, env, intrinsicGas, txTracer)) return;
 
-            if (!ValidateDeltaGasLimit(entry, env, txTracer))
-            {
-                return;
-            }
+            if (!ValidateDeltaGasLimit(entry, env, txTracer)) return;
 
             if (!_stateProvider.AccountExists(env.Sender))
-            {
                 if (env.GasPrice == UInt256.Zero)
-                {
                     _stateProvider.CreateAccount(env.Sender, UInt256.Zero);
-                }
-            }
 
-            if (!ValidateSenderBalance(entry, env, intrinsicGas, txTracer))
-            {
-                return;
-            }
+            if (!ValidateSenderBalance(entry, env, intrinsicGas, txTracer)) return;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4562326)